--- models.py.orig      2014-07-08 17:49:37.000000000 +0200
+++ models.py   2014-07-08 17:51:07.000000000 +0200
@@ -780,7 +780,7 @@
     # This code assumes that
     # self.user.email() == users.get_current_user().email()
     current_user = users.get_current_user()
-    if self.user.user_id() != current_user.user_id():
+    if self.user.id != current_user.id:
       # Mainly for Google Account plus conversion.
       logging.info('Updating user_id for %s from %s to %s' % (
         self.user.email(), self.user.user_id(), current_user.user_id()))
